packages/rocketchat-ui-message/client/messageBox.html

```diff
@@ -39,6 +39,16 @@
 					</div>
 					<textarea autofocus name="msg" maxlength="{{maxMessageLength}}" placeholder="{{_ 'Message'}}" rows="1" class="rc-message-box__textarea js-input-message autogrow-short"></textarea>
 					{{#unless disableSendIcon}}
+						{{#if hasOembed}}
+							<div class="rc-message-box__embed">
```

Comment:
Can we add a tooltip here? That would make the UX far more obvious, I feel.
Just add the class `rc-tooltip` and the property `aria-label` with the tooltip text.

Reply:
What should I write in the label? Thought of "Link preview is enabled/disabled for this message"

Reply:
That would be fine, perhaps a bit less verbose.